lo.js

```diff
@@ -5,9 +5,6 @@ const chalk = require('chalk');
 const debug = require('debug')('lo');
 const yargs = require('yargs');
 
-// Needed because the cognito library tries to fetch the user-agent from the browser
-global.navigator = () => null;
```

Comment:
From what I can tell, the Cognito library referenced here was removed [here](https://bitbucket.org/lifeomic/lifeomic-cli/commits/b2bead0aa8581c674544f6364a2d17424f254f90). I suspect this was just never updated.

Reply:
For context, this line breaks in Node 22 because `global.navigator` is immutable.